subsys/sd/sd_ops.c

```diff
@@ -756,6 +756,11 @@ int card_write_blocks(struct sd_card *card, const uint8_t *wbuf, uint32_t start_
 /* IO Control handler for SD MMC */
 int card_ioctl(struct sd_card *card, uint8_t cmd, void *buf)
 {
+	int ret = k_mutex_lock(&card->lock, K_MSEC(CONFIG_SD_LOCK_TIMEOUT));
+	if (ret) {
```

Comment:
Could you add an explicit check? I know this is just my opinion but everything that is not a bool should be explicitly checked for a value.

Reply:
I aligned it with the style of the rest of the file.

Reply:
as discussed ;)